src/style.css

```diff
@@ -0,0 +1,134 @@
+html,
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+
+  font-family: Roboto;
+  font-size: 11px;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 13px;
+  color: #000;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
```

Comment:
add display flex to card, and center elements inside card with flex properties, not margins/paddings